### PR TITLE
[14.0][ADD]Added Module account_move_line_menu

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

**Review by dreispt — APPROVED** [review]
👍🏼

File: account_move_line_menu/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+# from . import models
```
Comment (bodedra) [inline account_move_line_menu/__init__.py]: Remove commented code.

File: account_move_line_menu/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Menu option available at Accounting > Accounting > Miscellaneous > Journal items.
```
Comment (bodedra) [inline account_move_line_menu/readme/USAGE.rst:3]: non-blocking comment. Enterprise edition has `Accounting` menu and community edition has `Invoicing` menu.

Comment (dreispt) [reply account_move_line_menu/readme/USAGE.rst:3]: Good point, better add both cases.

**Review by bodedra — APPROVED** [review]
Functional reviewed. :+1:

**Pull request merged**